ansible/roles/products/hand-e/server/desktop-icons/templates/scripts/launch-everything-server.j2

```diff
@@ -0,0 +1,8 @@
+#jinja2: trim_blocks:False
```

Comment:
please use conditional waiting as here for example:
https://github.com/shadow-robot/aurora/blob/f1f8f9fcd499ef01497642f79e8caaf5364cc599/ansible/roles/products/hand-e/server/desktop-icons/templates/scripts/launch-everything-bimanual-arm-hand.j2

Reply:
done in the last commit